src/screens/__snapshots__/LoginScreen.spec.tsx.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Contributors should render the Welcome Screen 1`] = `
+<div>
```

Comment:
These snapshots don't look correct. Can they be regenerated?

Reply:
I will have to rewrite the test files. I will get them done by Friday.

Reply:
No rush

Reply:
I will need more time to write the correct jest test files. The ones I am rewriting are failing the test and I am unsure how to correct them. I will go through the jest documents and get them done by the end of next week.

Reply:
As long as your tests run correctly locally, I would suggest removing and not committing these snapshot files, so we can approve the rest of this PR. Pending @MelisaIm approval

Reply:
Resolved the test file issues. The snapshots are looking good to me now. Please take a look.